### PR TITLE
docs: TraceConfig and context attributes

### PR DESCRIPTION
**Pull request by fjcasti1**

### CODE REVIEW

File: python/instrumentation/openinference-instrumentation-bedrock/README.md [hunk 1]
```diff
@@ -129,3 +129,7 @@ print(out.get("content")[-1].get("text"))
 ## More Info
 
 More details about tracing with OpenInference and `phoenix` can be found in the [`phoenix` documentation](https://docs.arize.com/phoenix).
+
+Learn how to customize spans [here](../../openinference-instrumentation/README.md#customizing-spans).
+
+Learn how to configure tracing [here](../../openinference-instrumentation/README.md#tracing-configuration).
```
Comment (mikeldking) [inline python/instrumentation/openinference-instrumentation-bedrock/README.md:135]: maybe we make this a bulleted list:
- More info on OpenInference and Phoenix
- How to customize spans to track sessions, metadata, etc.
- How to account for private information and span payload customization

Comment (fjcasti1) [reply python/instrumentation/openinference-instrumentation-bedrock/README.md:135]: Yep, that'd be better

File: python/openinference-instrumentation/README.md [hunk 2]
```diff
@@ -8,3 +9,72 @@ Utility functions for OpenInference instrumentation.
 ```shell
 pip install openinference-instrumentation
 ```
+
+## Customizing Spans
+
+This core `openinference-instrumentation` package offers convenient tools to read & write from the OTEL Context. The OTEL context is then used to propagate attributes to every span created. We do this using Python context managers:
+
+* `using_session`: to specifiy a session ID.
```
Comment (mikeldking) [inline python/openinference-instrumentation/README.md:17]:
```suggestion
* `using_session`: to specifiy a session ID to track multi-turn conversations with a user
```

File: python/openinference-instrumentation/README.md [hunk 3]
```diff
@@ -8,3 +9,72 @@ Utility functions for OpenInference instrumentation.
...
+* `using_user`: to specifiy a user ID.
+* `using_metadata`: to add custom metadata.
+* `using_tag`: to add tags.
+* `using_prompt_template`: to reflect the prompt template used.
+* `using_attributes`: to specify all the above at once.
```
Comment (mikeldking) [inline python/openinference-instrumentation/README.md]: give the rationale as to why you would want to set these. Readers might not know why or how they might want to use these

File: python/openinference-instrumentation/README.md [hunk 4]
```diff
@@ -8,3 +9,72 @@ Utility functions for OpenInference instrumentation.
...
+  
+For example:
+```python
+from openinference.instrumentation import using_attributes
+tags = ["tag_1", "tag_2", ...]
+metadata = {
+    "key-1": value_1,
+    "key-2": value_2,
```
Comment (mikeldking) [inline python/openinference-instrumentation/README.md:30]: use real-life examples - it's more powerful
```suggestion
tags = ["business_critical", "topic: weather", ...]
metadata = {
    "country": "United States",
```

File: python/openinference-instrumentation/README.md [hunk 5]
```diff
@@ -8,3 +9,72 @@ Utility functions for OpenInference instrumentation.
...
+    # "llm.prompt_template.version " = "v1.0"
+    ...
+```
+You can read more about this in our [docs](https://docs.arize.com/phoenix/tracing/how-to-tracing/customize-spans).
+
+
+## Tracing Configuration
+
+
+The core `openinference-instrumentation` package offers the `TraceConfig` class, which helps set up privacy configurations to mask specific fields in your spans. For instance, you may want to keep sensitive information from being logged for security reasons, or you may want to limit the size of the base64 encoded images logged to reduced payload size.
```
Comment (mikeldking) [inline python/openinference-instrumentation/README.md:61]:
```suggestion
This package contains the central `TraceConfig` class, which lets you specify a tracing configuration that let's you control things like data privacy and payload sizes. For instance, you may want to keep sensitive information from being logged for security reasons, or you may want to limit the size of the base64 encoded images logged to reduced payload size.
```

**Review by mikeldking — APPROVED** [review]

File: python/instrumentation/openinference-instrumentation-bedrock/README.md (hunk 1 above)
Comment (fjcasti1) [reply python/instrumentation/openinference-instrumentation-bedrock/README.md]: Done

File: python/openinference-instrumentation/README.md (hunk 3 above)
Comment (fjcasti1) [reply python/openinference-instrumentation/README.md]: Done

**Pull request merged**